packages/cloud-cognitive/src/components/ExpressiveCard/ExpressiveCard.stories.js

```diff
@@ -120,7 +120,8 @@ WithActionIcon.args = {
   actionIcons: [
     {
       id: '1',
-      icon: <ArrowRight24 onClick={() => {}} />,
+      icon: ArrowRight24,
```

Comment:
We should add an `iconDescription` prop here as well since that is something that is used in the button [here](https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/560/files#diff-8ef3dc50c7bfa3e42e6d8aec034961f6e564544c5e691af803b7408512ec81a6R75).

Reply:
Nevermind, that's productive only I'm just seeing ^

Reply:
you're only half wrong here haha. i did need to add an `aria-label` prop to the icon, which i've updated 👍

Reply:
haha ok, glad that worked out